pkg/cluster/meta.go

```diff
@@ -68,15 +68,15 @@ func WithAnnotations(annotationKeyValue ...string) MetaOptions {
 	}
 }
 
-func extractKeyValues(kv []string) (map[string]string, error) {
-	lenKV := len(kv)
+func extractKeyValues(keyValues []string) (map[string]string, error) {
+	lenKV := len(keyValues)
 	if lenKV%2 != 0 {
 		return nil, fmt.Errorf("passed elements should be in key/value pairs, but got %d elements", lenKV)
 	}
 
-	kvMap := make(map[string]string, lenKV%2)
+	kvMap := make(map[string]string)
```

Comment:
i was thinking, and probably is still 
- so lenKV%2 should be correct to lenKV/2 ,which was a mistake
- but isnt that pre-allocate memory is better than dynamically add one by one, even,it is a small set of key-value from label/annotation